tools/infrasys/src/root.rs

```diff
@@ -108,11 +124,16 @@ fn add_keys_kms(
             num_keys: (*available_keys).len(),
         }
     );
-
+    let default_region = thread::spawn(move || {
+        let runtime = tokio::runtime::Runtime::new().context(error::RuntimeCreationSnafu)?;
+        runtime.block_on(async_get_region())
+    })
+    .join()
+    .map_err(|_| error::Error::ThreadJoin {})??;
```

Comment:
Can we abstract this into `get_region` that's called from `add_keys` and `create_root`? Or alternatively, handle it once before calling either function, since otherwise there's potential for this to blow up on refactoring if another tokio runtime has been started.

Reply:
fixed